misk/src/main/kotlin/misk/metrics/backends/signalfx/SignalFxBackendModule.kt

```diff
@@ -0,0 +1,32 @@
+package misk.metrics.backends.signalfx
+
+import com.codahale.metrics.MetricRegistry
+import com.google.common.util.concurrent.Service
+import com.google.inject.Provides
+import com.signalfx.codahale.reporter.SignalFxReporter
+import com.signalfx.metrics.auth.StaticAuthToken
+import misk.config.AppName
+import misk.inject.KAbstractModule
+import misk.inject.addMultibinderBinding
+import misk.inject.to
+import javax.inject.Singleton
+
+class SignalFxBackendModule : KAbstractModule(){
+    override fun configure() {
+        binder().addMultibinderBinding<Service>().to<SignalFxReporterService>()
+    }
+
+    @Provides
+    @Singleton
+    fun signalFxReporter(
+            @AppName appName: String,
+            config: SignalFxBackendConfig,
+            metricRegistry: MetricRegistry
+            ): SignalFxReporter {
```

Comment:
nit: this indentation seems weird - are you using the standard codestyle from square/codestyles?

Reply:
I am. Not sure what happened here, but cleaned it up.